packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFGeneratedWebKitApis.h

```diff
@@ -406,8 +408,9 @@ NSObject<FlutterMessageCodec> *FWFWKWebViewConfigurationFlutterApiGetCodec(void)
 @interface FWFWKWebViewConfigurationFlutterApi : NSObject
 - (instancetype)initWithBinaryMessenger:(id<FlutterBinaryMessenger>)binaryMessenger;
 - (void)createWithIdentifier:(NSNumber *)identifier
-                  completion:(void (^)(NSError *_Nullable))completion;
```

Comment:
These are going to require code changes. For simplicity of unblocking your end goal, can you just update to Pigeon `^7.0.5`?

Reply:
Done